packages/story-editor/src/app/media/utils/getResourceFromLocalFile.js

```diff
@@ -154,6 +158,14 @@ const getResourceFromLocalFile = async (file) => {
     // Not interested in the error here.
     // We simply fall back to the placeholder resource.
   }
+  const { src, poster } = resource;
+  const imageSrc = type === 'image' ? src : poster;
+  try {
+    const color = await getMediaBaseColor(imageSrc);
```

Comment:
We should use `getTimeTracker` in `getMediaBaseColor` to measure how long it takes to get the color of an image.

Reply:
I did some testing and results were very interesting!. 

Generated base color from blob. 
Image - 177.85000002384186
Video - 428.8799999952316

Generated base color from image. 
Image - 1494.105 / 1360.30
Video - 652.7250000238419

Worth noting, that used the same image / video for both. I could do more testing, but results are positive.

Reply:
That's in milliseconds I presume?

Can you add the `getTimeTracker` telemetry in `getMediaBaseColor` in this PR?

Reply:
Sure!

Reply:
I add time tracking at the base color generation. I saw one example, that was 33ms compared to 532ms on the same video. So much better!

Reply:
Removed after feedback.